tests/app/shared/issue/title/title.component.spec.ts

```diff
@@ -0,0 +1,66 @@
+import { TitleComponent } from '../../../../../src/app/shared/issue/title/title.component';
+import { ISSUE_WITH_EMPTY_DESCRIPTION } from '../../../../constants/githubissue.constants';
+import { Issue } from '../../../../../src/app/core/models/issue.model';
+import { FormBuilder, NgForm } from '@angular/forms';
+import { of } from 'rxjs';
+
+describe('TitleComponent', () => {
+  let titleComponent: TitleComponent;
+  let issueService: any;
+  let thisIssue: Issue;
+  let formBuilder: any;
+
+  let form: any;
+  let formResetForm: any;
+
+  beforeEach(() => {
+    formBuilder = new FormBuilder();
+
+    form = new NgForm([], []);
+    formResetForm = spyOn(form, 'resetForm');
```

Comment:
Will it be better to shift both `form` and `formResetForm`  into the scope of the `updateTitle` test, since they are not used elsewhere?

Reply:
Agreed, that definitely would be better 👍